### DISCUSSION TITLE
Migrate from Micro$oft ₲itHub

### DISCUSSION THREAD

**Discussion by KOLANICH** [open-source-ideas/ideas#370] [Ideas]
Micro$oft-owned ₲itHub [has discriminated users of 1FA passwords while having commercial interest in success and widespread adoption of FIDO 1FA specifications and Windows Hello implementation which it promotes as a replacement for passwords](https://codeberg.org/KOLANICH/Fuck-GuanTEEnomo). It may make sense to leave ₲itHub.

Of course it may make sense to bidirectionally synchronize issues and comments in repos on different platforms, but there is no free (as in "freedom") and open source solution for that now, and there is no free (as in "free beer") solution for that too. https://github.com/open-source-ideas/ideas/issues/369

**Comment by FredrikAugust** [comment]
If you want to host a mirror of this repository somewhere else you're more than free to do so! I don't think we'll move from GitHub anytime soon though due to the widespread adoption, and it being the de-facto place for open source contribution.

**Reply by KaKi87** [reply]
> de-facto place for open source contribution

That's debatable.

In fact, none of the biggest open source projects in existence are maintained on GitHub.
